pallets/phala/src/compute/wrapped_balances.rs

```diff
@@ -89,6 +92,18 @@ pub mod pallet {
 	pub type StakerAccounts<T: Config> =
 		StorageMap<_, Twox64Concat, T::AccountId, FinanceAccount<BalanceOf<T>>>;
 
+	#[pallet::storage]
+	#[pallet::getter(fn election_locks)]
+	pub type ElectionLocks<T: Config> = StorageMap<_, Twox64Concat, T::AccountId, BalanceOf<T>, ValueQuery>;
+
+	#[pallet::storage]
```

Comment:
```suggestion
	/// The WPHA a user owes the system because of a lack of liquid token. Wills be settled by `slash_reserved()` in the future.
	#[pallet::storage]
```